livesync/indico_livesync/uploader.py

```diff
@@ -48,17 +48,17 @@ def run(self, records):
             self.processed_records(batch)
         self.logger.info('%s finished', self_name)
 
-    def run_initial(self, events, total=None):
+    def run_initial(self, records, total=None):
```

Comment:
I think we can make `total` mandatory..

Reply:
Previously I would say we could want to run an export without progress. 
But right now, I don't see any cases for making it optional either.

Reply:
yeah, I think if we ever find a reason why we wouldn't want a progress display, that should be made explicit through a new CLI arg. but it doesn't seem particularly useful...